### DISCUSSION TITLE
support: no errors, but video doesn't play

### DISCUSSION THREAD

**Discussion by ohthehugemanatee** [ressu/kube-plex#21] [General]
Got it running with an NFS-mounted `/transcode` directory. Transcode pod is started correctly, and `/transcode` fills up as the transcoder runs. No problems reported in the console. But... the video is stuck on buffering at 0:00.

What is the desired flow, here? Console suggests that the transcoded segments are sent back to the master with a PUT request. Is that right? Or does the master just read the shared `/transcode` directory? Any advice on debugging further would be appreciated.

``` values.yaml
kubePlex:
  image:
    repository: ghcr.io/ressu/kube-plex
    tag: latest
    pullPolicy: Always


timezone: Europe/Berlin

ingress:
  enabled: true
  hosts:
    - plex.vert
    - plex.example.com

nodeSelector:
  beta.kubernetes.io/arch: amd64

persistence:
  transcode:
    enabled: true
    accessMode: ReadWriteMany
  data:
    claimName: "nfs-claim"
    subPath: "media"

```

**Comment by ressu** [comment]
Plex reads the files directly from `/transcode` while the transcode process sends status updates back to the main process via a POST request (I think it was POST). If you are seeing dark screen only, the Plex process is most likely unable to see the files on the shared directory.

Looking at your configuration, I'm not quite sure what your transcode PVC looks like. I think the default in the template is an `emptyDir` which isn't shared between processes, so that could be the cause...
